### PR TITLE
[PLAY-7] Shadow section for visual guidelines

### PR DESCRIPTION
**Pull request by nickamantia**
#### Screens

[INSERT SCREENSHOT]
Screen Shot 2021-11-17 at 3 03 35 PM

Screen Shot 2021-11-17 at 3 03 47 PM

#### Breaking Changes

[Yes/No (Explain)]

#### Runway Ticket URL

[INSERT URL]
https://nitro.powerhrg.com/runway/backlog_items/PLAY-7

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Comment by jasoncypret** [comment]
@nickamantia We don't have global props for shadows. Here's an example of where I think you will land
Shadows

**Comment by nickamantia** [comment]
Screen Shot 2021-11-18 at 3 47 11 PM

**Review by jasoncypret — APPROVED** [review]

**Comment by nickamantia** [comment]
0138A8E2-AFB6-4B06-A90D-9F88754A9420_1_105_c

File: playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<%= pb_rails("title", props: {margin_top: "xl", text: "Shadows", tag: "h1", size: 1 }) %>
+
+<%= pb_rails("body", props: { margin_y: "md"} ) do %>
+  Shadows are a night way of showing depth. Modals, buttons, & popovers are examples of items that need to have a shadow to indicate their position relative to others. Our tokens help you expand those to the other custom elements you build using playbook.
+<% end %>
+
+<br>
+
+<h4 aria-label="online" class="pb_title_kit_4 pt_sm">Tokens</h4>
```
Comment (thestephenmarshall) [inline playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb:9]: Wondering why is this not using our `Title` kit?

File: playbook-website/app/views/pages/visual_guidelines_partials/_shadow.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<%= render :partial => "pages/visual_guidelines_partials/pb_doc_shadow",
+locals:{ title: "Shadow Options",
+colors: [{name:"Shadow None",variable:"shadow_none"},
+  {name:"Shadow Deep",variable:"shadow_deep"},
+  {name:"Shadow Deeper",variable:"shadow_deeper"},
+  {name:"Shadow Deepest",variable:"shadow_deepest"}]} %>
```
Comment (thestephenmarshall) [inline playbook-website/app/views/pages/visual_guidelines_partials/_shadow.html.erb:6]:
```suggestion
  {name:"Shadow Deepest",variable:"shadow_deepest"}]} %>
  
```

File: playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb (hunk 1 above)
Comment (nickamantia) [reply playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb:9]: Oops sorry! I'll switch that to the Title kit. Thank you!

**Review by thestephenmarshall — APPROVED** [review]

File: playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb (hunk 1 above)
Comment (nickamantia) [reply playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb]: Fixed!  Replaced the h4 with the Title kit!

File: playbook-website/app/javascript/site_styles/_shadow_guidelines.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    box-shadow: $shadow_none;
+}
+
+.shadow-box-deep { 
+    box-shadow: $shadow_deep;
+}
+
+.shadow-box-deeper {
+    box-shadow: $shadow_deeper;
+
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_shadow_guidelines.scss:22]:
```suggestion
```

File: playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                <% end %>
+             
+                <%= pb_rails("body", props: {classname: "flex-item", padding_right: "sm", margin_left: "sm" }) do %>
+                  <%= pb_rails("caption", props: { text: "value: 0 4px 10px 0 rgb(60 106 172 / 16%)", size: "xs", padding_right: "xs" }) %>
+                  <% end %>
+            <% end %>
+            <% end %>
+            
+            <br>
+
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb:35]: Take a look at these files:
https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_border_radius.html.erb
https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_max_width.html.erb

I'd like to clean up this view. You can make your kits settings exactly like these files including margin, padding, ect.

File: playbook-website/app/views/pages/visual_guidelines_partials/_shadow.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<%= render :partial => "pages/visual_guidelines_partials/pb_doc_shadow",
+locals:{ title: "Shadow Options",
+colors: [{name:"Shadow None",variable:"shadow_none"},
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_shadow.html.erb:3]: I'd like to move these settings to the controller. See this example: 

https://github.com/powerhome/playbook/blob/master/playbook-website/app/controllers/pages_controller.rb#L85

File: playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+            <br>
+           
+      <% react_contents = Rails.root.join("app/views/pages/code_snippets/shadow_erb.txt").read %>
+    
+      <%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper" }) do %>
+        <%= pb_rails("caption", props: { padding_x: "md", padding_top: "md", text: "Example in Sass" }) %>
+          <div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+            <pre class="highlight"><%= raw rouge(react_contents, "react") %></pre>
+          </div>        
+      <% end %>
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb:78]: I'd like to move this variable to the controller. See this example: https://github.com/powerhome/playbook/blob/master/playbook-website/app/controllers/pages_controller.rb#L85

Also, note the subtle changes in this code example. You might consider copying this: https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_border_radius.html.erb#L49-L54

**Review by jasoncypret — CHANGES_REQUESTED** [review]
Just some small changes

**Review by jasperfurniss — APPROVED** [review]

**Review by jasoncypret — APPROVED** [review]

**Pull request merged**